src/runtime/index.css

```diff
@@ -1,3 +1,4 @@
+@import '#build/ui.css';
```

Comment:
Just to make sure I understand Tailwind correctly:
If any of `--color-{color}-{shade}` vars don't get used by developers then Tailwind wont include them in the final build right? Otherwise this will effectively create duplicates and should be on opt-in feature IMO.

Although I do agree this should be automated. Considering that it was just a copy-paste for those interested in having them.

Reply:
Yes, this only tells Tailwind CSS to generate utilities such as `text-primary-500`, etc. like it did with the default theme like `text-indigo-500` for example. However, the classes will only be included in the bundle if written by the user since we're not using them in our theme.

I'm also using `@theme default`, so users can override them if they want in their own `@theme`.